code/modules/vehicles/mecha/working/ripley.dm

```diff
@@ -138,6 +138,21 @@
 	var/obj/item/mecha_parts/mecha_equipment/mining_scanner/scanner = new
 	scanner.attach(src)
 
+/obj/vehicle/sealed/mecha/working/ripley/cargo
+	desc = "An ailing, old, repurposed cargo hauler."
```

Comment:
you should probably directly mention the reduced equipment capacity in this description

Reply:
You're right.